package.json

```
@@ -18,7 +18,37 @@
         "url": "git+https://github.com/nick-w-nick/eslint-config-lintification.git"
     },
     "homepage": "https://github.com/nick-w-nick/eslint-config-lintification#readme",
-    "keywords": ["eslint", "lintification", "lint", "typescript", "nodejs", "backend"],
+    "keywords": [
+        "eslint",
```

@beanc16 You think these are worth adding?

```suggestion
        "eslint",
        "eslint-config",
        "config",
```

---

Perhaps with config omitted?
```suggestion
        "eslint",
        "eslint-config",
```

---

Works for me!